lib/util/state.js

```diff
@@ -581,17 +581,28 @@ function getItineraryToRender(state) {
 const routeSelector = (state) => Object.values(state.otp.transitIndex.routes)
 const routeViewerFilterSelector = (state) => state.otp.ui.routeViewer.filter
 
+const getAgencyIdForFilter = (name, transitOperators) => {
+  const operators = transitOperators.filter(
+    (operator) => operator.name === name
+  )
+  return operators.map((x) => x.agencyId)
```

Comment:
Probably cleaner to do this all as one call! Then we don't need to assign the variable

Reply:
Also the name doesn't make it clear that this will return an array!

Reply:
Good call! Let me know if changing it to `getAgencyIdsForFilter` works okay or if it could be more descriptive!